serializer/serializer.c

```diff
@@ -761,3 +761,51 @@ status_t send_mam_req_deserialize(const char* const obj, ta_send_mam_req_t* req)
   cJSON_Delete(json_obj);
   return ret;
 }
+
+status_t mqtt_device_id_deserialize(const char* const obj, char* device_id) {
+  if (obj == NULL) {
+    log_error(seri_logger_id, "[%s:%d:%s]\n", __func__, __LINE__, "SC_SERIALIZER_NULL");
```

Comment:
How about defining new macro which utilizes `__func__` and `__LINE__` for `log_error`?

Reply:
I will open the other PR for this.

Reply:
```
#define SCYLLA_LOG_ERROR(fmt, ...) \
  logger_helper_print(scylla_api_logger_id, LOGGER_ERR, "[%s:%d:]" fmt, __func__, __LINE__, __VA_ARGS__)
#define SCYLLA_LOG_DEBUG(fmt, ...) \
  logger_helper_print(scylla_api_logger_id, LOGGER_DEBUG, "[%s:%d:]" fmt, __func__, __LINE__, __VA_ARGS__)

```
I do this when writing Scylla interface. Not sure whether it could be better.